crates/bevy_render/src/color/mod.rs

```diff
@@ -1083,69 +1083,184 @@ impl Color {
             }
         }
     }
-}
 
-impl Default for Color {
-    fn default() -> Self {
-        Color::WHITE
+    /// New `Color` from `[f32; 4]` in sRGB colorspace.
```

Comment:
These comments are now outdated.

Reply:
Not sure about docs. Should it be now: "New `Color` from `impl Into<[f32; 4]>` in sRGB colorspace"?

Reply:
"New Color from [f32; 4] (or a type that be converted into them) in sRGB colorspace"